packages/ui-kit/src/components/ErrorFallback/ErrorFallback.tsx

```diff
@@ -29,17 +29,13 @@ const Icon = ({ color }: { color?: string }) => (
 )
 
 export const ErrorFallback = React.forwardRef<HTMLDivElement, ErrorFallbackProps>(
-  ({ error = serverErrorMessage, className, style, baseTheme, color, ...rest }, forwardedRef) => {
+  ({ error = serverErrorMessage, className, baseTheme, color, ...rest }, forwardedRef) => {
     const { componentContainer, setRef } = useForwardedRefCallback(forwardedRef)
     useSetupTheme({ componentContainer, baseTheme })
 
     return (
       <div ref={setRef} className={componentStyles.rootErrorFallback} {...rest}>
-        <div
-          className={classnames(componentStyles.container, className)}
-          style={{ ...style }}
-          data-testid="error-fallback-container"
-        >
+        <div className={classnames(componentStyles.container, className)} data-testid="error-fallback-container">
```

Comment:
We should use top-level elements for defining styles to use simple CSS-selectors, e.g. `.errorFallback div svg { }`. With the current change `styles` goes [here](https://github.com/propeldata/ui-kit/pull/194/files#diff-adb814adda884a78653dc385d0aff917af728d9bd23e524e19ddc43b05a6d0efL37) (as part of `{...rest}`).
In fact, I'll do the same for `className`.

Reply:
In this case I find it necessary to pass the `style` prop to that inner `div` element, actually, we could get rid of that outer `div` element, it looks weird that we have `div` nested under a `div`.

```ts
<div
  ref={setRef}
  className={classnames(componentStyles.rootErrorFallback, className)}
  {...rest}
  data-testid="error-fallback-container"
>
  <div className={componentStyles.container}> // this `div` is the only child of the top level `div`
    <Icon color={color} />
    {error && (
      <>
        <p role="alert" aria-live="assertive">
          {error.title}
        </p>
        <p role="alert" aria-live="assertive">
          {error.body}
        </p>
      </>
    )}
  </div>
</div>
```

In case removing the top level `div` would break anything please at least keep the `style` prop in the inner one as some props would not work in the inner one while trying to select e.g the icon or the text.